docs/authentication/certificate-service-authentication.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: default
+title: Certificate service Authentication
+parent: Authentication
+nav_order: 2
+permalink: /authentication/certificate-service
+---
+
+# OAuth Authentication
+{: .no_toc }
+
+## Table of contents
+{: .no_toc .text-delta }
+
+1. TOC
+   {:toc}
+
+---
+
+## Description
+
+Certificate service Authentication is a class in the context of Alert Notification service node client. It helps you to authenticate directly against Alert Notification service. 
```

Comment:
CertificateAuthentication is a class in the context of Alert Notification service node client. It is used to trigger the mTLS based authentication, you just need to provide the certificate and private key generated from your Alert Notification service instance (e.g. from a Service Key or Service Binding). 

Also add a link to help.sap.com for further reading